glide.yaml

```diff
@@ -4,6 +4,7 @@ import:
   verison: 496d495156da218b9912f03dfa7df7f80fbd8cc3
 - package: github.com/fabric8io/golang-jenkins
   verison: fd49fcd723578afb98fd72961f0c0074393ae97a
```

Comment:
Ah this is why the sha wan't getting the right version and using the latest commit on master.  Thanks @jimmidyson.